plugins/search/src/components/SearchModal/SearchModal.tsx

```diff
@@ -144,14 +146,20 @@ export const Modal = ({ open = true, toggleModal }: SearchModalProps) => {
             </List>
           )}
         </SearchResult>
+        {config.getOptionalBoolean('permission.enabled') ? (
+          <SearchRefineResultsPrompt />
+        ) : null}
       </DialogContent>
-      <DialogActions className={classes.dialogActionsContainer}>
-        <Grid container direction="row">
-          <Grid item xs={12}>
-            <SearchResultPager />
+
+      {!config.getOptionalBoolean('permission.enabled') ? (
```

Comment:
Slightly worried that this could be a bit of a trap. Both the config reading, in case we want to inject some more logic to determine whether permissions are enabled, but also the global switch to enable/disable permissions everywhere. Avoiding digging into solutions for now but keeping my eyes on this 👀  :grin: